Description: Stop sphinx trying to download from network at build time
  Sphinx tries to download catalogues for intersphinx links -- suppress that
  here by removing the intersphinx extension.
Author: Stuart Prescott <stuart@debian.org>
Index: translate-toolkit-3.3.3/docs/conf.py
===================================================================
--- translate-toolkit-3.3.3.orig/docs/conf.py
+++ translate-toolkit-3.3.3/docs/conf.py
@@ -39,7 +39,6 @@ extensions = [
     "sphinx.ext.autodoc",
     "sphinx.ext.coverage",
     "sphinx.ext.extlinks",
-    "sphinx.ext.intersphinx",
     "sphinx.ext.todo",
 ]
 
